ddp-workspace/projects/ddp-testboston/src/assets/i18n/zh.json

```diff
@@ -619,7 +619,10 @@
     "TooltipAlt": "信息",
     "SubjectPanel": {
       "Link": "返回受试者查找",
-      "Text": "受试者邀请代码："
+      "InvitationCode": "受试者邀请代码：",
+      "Email": "[ZH] Email:",
+      "Name": "[ZH] Name:",
+      "ShortId": "[ZH] Subject Short ID/HRUID:"
```

Comment:
Don't think this is hurting anything, but the subject panel will only be viewable by study administrators and while we do support a ton of different languages for testboston participants, it is OK (at this point) to show subject panel labels always in English.
At any rate, this is fine, though.

Reply:
yes, I realized it later but decided keeping this since the work was already done :)